### DISCUSSION TITLE
Exploring files using vim commands

### DISCUSSION THREAD

**Discussion by jerpint** [JetBrains/ideavim#282] [Q&A] [answered]
I'm wondering if it's possible to navigate the file tree seamlessly from code without touching the mouse. Similarly, can I browse the file tree using vim bindings, open files, etc., without ever touching the mouse? Similar to using nerdtree in vim

**Comment by AlexPl292** [comment, marked as answer]
Hi! Yes, you can. IdeaVim has built in NERDTree support: https://github.com/JetBrains/ideavim/wiki/Emulated-plugins#nerdtree
